cmd/guaccollect/cmd/deps_dev.go

```diff
@@ -77,17 +90,28 @@ var depsDevCmd = &cobra.Command{
 		if err != nil {
 			logger.Errorf("unable to register oci collector: %v", err)
 		}
+		if opts.enablePrometheus {
+			go func() {
+				http.Handle("/metrics", m.MetricsHandler())
+				if err := http.ListenAndServe(opts.promeAddr, nil); err != nil {
+					logger.Fatalf("Error starting HTTP server: %v", err)
```

Comment:
should failure of the metric server cause an exit of the service or just log an error?

Reply:
It should exit, if not people will assume the server is running.

Reply:
I think it makes to fail here as you are specificying that you want to enable the metrics server. If it does not succeed, you would want to exit with an error